jaeger-core/src/main/java/com/uber/jaeger/samplers/ConstSampler.java

```diff
@@ -65,9 +66,17 @@ public boolean equals(Object other) {
     return false;
   }
 
+  @Override
+  public int hashCode() {
+    int result = (decision ? 1 : 0);
+    result = 31 * result + (tags != null ? tags.hashCode() : 0);
+    return result;
+  }
```

Comment:
can we not disable whatever error the plugging was giving? We do not need hashcodes in samplers.